<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Controller_settings</class>
 <widget class="QWidget" name="Controller_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройка микроконтроллера</string>
  </property>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="Settings">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>411</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Параметры</string>
   </property>
   <widget class="QGroupBox" name="FreqSet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Частота дискретизации</string>
    </property>
    <widget class="QRadioButton" name="freq10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>10 Гц</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="freq100">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>100 Гц</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="freq1000">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>1кГц</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
